### DISCUSSION TITLE
impossible to increase the page size limit

### DISCUSSION THREAD

**Discussion by TheLouD1** [requarks/wiki#7296] [Help / Questions] [answered]
hello, 
I've been hitting the error network errror 413. After a quick search, saw that it's caused not because of a network issue but because the page is too big and hit over the default 1mb size. 
Tried to find a solution, saw a lot of docs for nginx and other proxy, which is not my problem as the wiki I use has a local purpose only 

Saw this [reddit post](https://www.reddit.com/r/selfhosted/comments/ovoj1w/wikijs_giving_status_code_413_when_saving_note/) that explained what they did to fix + reddirected to wiki.js' github, specifically to [this issue](https://github.com/Requarks/wiki/issues/2797) 
followed the link the staff replied with. It's the same as the line referred to on reddit. Awesome.
The line is a bit different (it nows has a || element) but it's still overall the same line and same effect. 
So I tried it, and well.. disappointment ensured as the swap isn't effective (moved from 1 mb to 20 but nothing changed

![image](https://github.com/user-attachments/assets/389ef6c3-f74f-43c2-9745-eb92a57e18be)

before this, I tried to increase the upload limit in the management page, under the security tab, but sadly no success : 
mine is set to 50mb with 10 attachements and well I've hit a 413 again with a pages that contains 7 pictures pasted in individual diagrams (as I can't paste directly within the body
each pic is small, the heaviest one being 235ko

any idea what could have gone wrong ?
I tried to reboot before and after mapping the /wiki/server on my docker compose, still no success

thanks

**Comment by NGPixel** [comment, marked as answer]
You need to edit the config.yaml file and change the bodyParserLimit. The `20mb` is a fallback and will be ignored when bodyParserLimit is already defined.
